lib/l10n_utils/dotlang.py

```diff
@@ -108,7 +105,7 @@ def translate(text, files):
     if lang == settings.LANGUAGE_CODE:
         return Markup(text)
 
-    tweaked_text = tweak_message(text)
+    tweaked_text = strip_whitespace(text)
```

Comment:
Good catch. This was a change from an earlier iteration of my attempt. I'll update.